codegen/codegen.cabal

```diff
@@ -0,0 +1,23 @@
+cabal-version:       2.2
+name:                codegen
+version:             0.1.0.0
+-- synopsis:
+-- description:
+homepage:            https://github.com/patrickt/codegen#readme
+license:             BSD-3-Clause
+author:              Patrick Thomson
+maintainer:          patrickt@github.com
+copyright:           Patrick Thomson
+category:            Web
+build-type:          Simple
+
+executable codegen
```

Comment:
Should this be a separate package, or should we make it a separate executable component in the `haskell-tree-sitter` package? If it _does_ belong in a separate package, can we rename the package to `haskell-tree-sitter-codegen` or something?

Reply:
Aha, we can actually make this an executable component in the haskell-tree-sitter package. Cool, that simplifies a ton of things.

Reply:
@patrickt: Looks like this didn’t happen before merge?